app/resources/api/v2/request_resource.rb

```diff
@@ -66,6 +67,10 @@ def pre_capture_pool_id
         _model.pooled_request&.pre_capture_pool_id
       end
 
+      def library_type
```

Comment:
You can also use try which returns nil if it does not exist e.g. `_model.try(:library_type)`